src/main/java/com/clozex/shop/model/Category.java

```diff
@@ -0,0 +1,39 @@
+package com.clozex.shop.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.SQLRestriction;
+
+@SQLDelete(sql = "UPDATE categories SET is_deleted = TRUE WHERE id = ?")
+@SQLRestriction(value = "is_Deleted = FALSE")
+@Table(name = "categories")
+@NoArgsConstructor
+@Getter
+@Setter
+@Entity
+public class Category {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private String name;
+
+    private String description;
+
+    @Column(nullable = false)
+    private Boolean isDeleted = false;
```

Comment:
better use primitive data type 
```suggestion
    private boolean isDeleted = false;
```